### PR TITLE
[CFP-73] Migrate user forms to GOVUK design system

### PR DESCRIPTION
**Pull request by naseberry**
#### What
Migrate certification form to the GOVUK design system form builder

#### Ticket
[Migrate user forms to the GOVUK design system form builder](https://dsdmoj.atlassian.net/browse/CFP-73)

#### Why
Part of a larger change to:

- continue migration to GOVUK Frontend
- resolve accessibility issues identified in the 2020 DAC report
- reduce then remove deprecated code

### CODE REVIEW

File: app/views/external_users/admin/external_users/_form.html.haml [hunk 1]
```diff
@@ -1,62 +1,50 @@
...
+          - if current_user.persona.provider.chamber?
+            = f.govuk_collection_radio_buttons :vat_registered,
+              [['Yes','true'],['No','false']],
+              :last,
+              :first,
+              inline: true,
+              legend: { text: t('.vat_registered'), size: 's' }
+
+    = f.govuk_text_field :supplier_number,
+      hint: { text: t('.supplier_number_hint') },
```
Comment (jsugarman) [inline app/views/external_users/admin/external_users/_form.html.haml:47]: do we need to worry about the safari bug here i.e. adding hints/`aria-describedby` impact on buttons?

File: spec/models/user_spec.rb [hunk 2]
```diff
@@ -39,7 +39,7 @@
   it { is_expected.to validate_length_of(:email).is_at_most(80).with_message('Email must be 80 characters or less') }
   it { is_expected.to allow_value('email@addresse.foo').for(:email) }
   it { is_expected.not_to allow_value('foo').for(:email) }
-  it { is_expected.to validate_length_of(:password).is_at_least(8).with_message('Email must be at least 8 characters') }
+  it { is_expected.to validate_length_of(:password).is_at_least(8).with_message('Password must be at least 8 characters') }
```
Comment (jsugarman) [inline spec/models/user_spec.rb:42]: oops, good catch

**Review by jsugarman — APPROVED** [review]
LGTM 👍  - one pointer regarding the recent safari bug and its impact

I guess we don't have much in the way of cucumber features to cover these but I may be looking to add some due to the sandbox bug

File: app/views/external_users/admin/external_users/_form.html.haml (hunk 1 above)
Comment (naseberry) [reply app/views/external_users/admin/external_users/_form.html.haml:47]: Not here @jsugarman as this radio collection has no hint.

**Pull request merged**